docs/hooks/enabling-hooks/drupal.md

```diff
@@ -0,0 +1,53 @@
+## Using hooks with Drupal
+
+https://www.drupal.org/docs/7/creating-custom-modules/understanding-the-hook-system-for-drupal-modules
```

Comment:
I think these two URLs are useful, but look a little messy as-is. Can you do the following before we merge? 

1. Put each link in a markdown hyperlink like: `[Understanding the hook system](https://www.drupal.org/docs/7/creating-custom-modules/understanding-the-hook-system-for-drupal-modules)`. (With our book settings, raw URLs don't become clickable without using this markdown)
1. Either put the links in a list with some text above the list like "Resources from Drupal", or put them in a short (like, one-sentence) paragraph to explain what they are. 

(Just so it looks nicer and is clickable.)